feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -30,8 +30,8 @@ import {
 } from '@spartacus/core';
 import {
   ICON_TYPE,
-  LAUNCH_CALLER,
   LaunchDialogService,
+  LAUNCH_CALLER,
```

Comment:
why move it to the end of list? the old sequence should be right

Reply:
when I save file, it automatic change the sequence